extensions/wikia/WikiFactory/DeadWikis/maintenance.php

```diff
@@ -382,6 +398,74 @@ private function disableDiscussion( $cityId ) {
 		}
 	}
 
+	// Gets the most recent post from the specified site and returns whether it was made less than 60 days ago
+	private function isActiveSite($site_id)
```

Comment:
You might want to run the MW auto-formatter on new code. You can find it in Confluence.